internal/action/globals.go

```diff
@@ -21,13 +21,6 @@ func WriteLog(s string) {
 	buffer.WriteLog(s)
 	if LogBufPane != nil {
 		LogBufPane.CursorEnd()
-		v := LogBufPane.GetView()
-		endY := buffer.LogBuf.End().Y
-
-		if endY > v.StartLine+v.Height {
```

Comment:
This check is redundant since `StartLine` has been already properly updated by `CursorEnd`.